README.md

```diff
@@ -10,55 +10,79 @@
+compliant OpenID Connect implementation. It allows consumers to access user details with consent and in the OpenID
+Connect UserInfo format.
 
 A typical workflow would be:
 
 1. Authenticate.
-2. Access a user-info resource via GET or POST request
+2. Access a user-info resource via GET or POST request. GET is recommended.
 
-All end points are User Restricted (see [authorisation](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation)). Versioning follows the API Platform standards (see [the reference guide](https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide)).
-User details data structures follow the OpenId Connect UserInfo specification (see [the specification](http://openid.net/specs/openid-connect-core-1_0.html#UserInfo))
+All end points are User Restricted (
+see [authorisation](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation)). Versioning follows the
+API Platform standards (
+see [the reference guide](https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide)).
+User details data structures follow the OpenId Connect UserInfo specification (
+see [the specification](http://openid.net/specs/openid-connect-core-1_0.html#UserInfo))
 
-You can dive deeper into the documentation in the [API Developer Hub](https://developer.service.hmrc.gov.uk/api-documentation/docs/api#openid-connect-userinfo).
+You can dive deeper into the documentation in
+the [API Developer Hub](https://developer.service.hmrc.gov.uk/api-documentation/docs/api#openid-connect-userinfo).
 
 <a name="how-to-build-and-test"></a>
+
 ## How to build and test
+
 Run the service `sbt run -Drun.mode=Dev`
 
 Run the tests & test coverage report `sbt clean coverage test it/test coverageReport`
 
 The test coverage report will be available in `target/scala-2.12/scoverage-report/index.html`
 
-Run the service in service manager; if you want live endpoints, then start dependencies thus: `sm2 --start AUTH DATASTREAM -r`
+Run the service in service manager; if you want live endpoints, then start dependencies thus:
+`sm2 --start AUTH DATASTREAM -r`
 
 Now you can test the sandbox `curl -v http://localhost:9836/sandbox/ -H 'Accept: application/vnd.hmrc.1.0+json'`
 
-Link to API documentation https://admin.qa.tax.service.gov.uk/api-catalogue/integrations/61b66a2b-a892-4197-bb40-eac67e2ce3c6/user-information  
-You need a devhub account from https://developer.qa.tax.service.gov.uk/developer/login and follow the instructions for making an app to subscribe to the test api on https://confluence.tools.tax.service.gov.uk/display/DTRG/Testing+an+API+microservice+on+Development+and+QA
+Link to API
+documentation https://admin.qa.tax.service.gov.uk/api-catalogue/integrations/61b66a2b-a892-4197-bb40-eac67e2ce3c6/user-information  
+You need a devhub account from https://developer.qa.tax.service.gov.uk/developer/login and follow the instructions for
+making an app to subscribe to the test api
```

Comment:
Add <dot> after this sentence.

Replace the reset of the section with...

Internal users may also reference this link on how to setup and test:
https://confluence.tools.tax.service.gov.uk/display/ApiPlatform/Testing+an+API+microservice+on+Development+and+QA

Internal reference documentation is also at:
https://confluence.tools.tax.service.gov.uk/display/GG/Scopes+and+Claims
https://confluence.tools.tax.service.gov.uk/display/GG/Userinfo+Endpoint

... and remove the rest of the remaining text.